src/Momento.Sdk/Config/Transport/IGrpcConfiguration.cs

```diff
@@ -42,6 +43,36 @@ public interface IGrpcConfiguration
     /// </remarks>
     public SocketsHttpHandlerOptions SocketsHttpHandlerOptions { get; }
```

Comment:
the keepalive settings are technically options set on the `SocketsHttpHandler` so they could be put into the [`SocketsHttpHandlerOptions`](https://github.com/momentohq/client-sdk-dotnet/blob/main/src/Momento.Sdk/Config/Transport/SocketsHttpHandlerOptions.cs) object instead if it seems cleaner to do it that way

Reply:
yeah that sounds like the right way to go to me

Reply:
Agreed. This could be confusing otherwise if someone is in a runtime that doesn't use `SocketsHttpHandler` but then they attempt to override this setting thinking it applies unconditionally.